gensound/transforms.py

```diff
@@ -190,31 +190,50 @@ def realise(self, audio):
 class Fade(Transform):
     min_fade = -50
     
-    def __init__(self, is_in=True, duration=3e3):
-        self.is_in = is_in
+    def __init__(self, duration=None, curve=None, is_in=True, degree=None):
```

Comment:
I think `is_in` should be before the other arguments, as it is more essential (also see comment below)

Reply:
Aren't we directing the user to use `FadeIn` and `FadeOut` and not `Fade` itself? 
I see the `Fade` class as a structural necessity for 
1. maintaining the two similar subclasses in one place
 2. backwards compatibility. 

But not  for user direct usage and calls.
We prefer the user to use `FadeIn()` rather than `Fade(is_in)`. Aren't we?

So, what params order you believe we should have?